packages/searchkit/src/__test__/core/AccessorManagerSpec.ts

```diff
@@ -91,113 +99,109 @@ describe("AccessorManager", ()=> {
+    const accessors = new AccessorManager()
+    const queryAccessor = new QueryAccessor('q')
     expect(accessors.add(queryAccessor)).toBe(queryAccessor)
     expect(accessors.getQueryAccessor()).toBe(queryAccessor)
     expect(accessors.getAccessors()).toEqual([queryAccessor])
-    expect(()=>accessors.add(queryAccessor)).toThrow(
-      "Only a single instance of BaseQueryAccessor is allowed"
+    expect(() => accessors.add(queryAccessor)).toThrow(
+      'Only a single instance of BaseQueryAccessor is allowed'
     )
 
     accessors.remove(queryAccessor)
     expect(accessors.getAccessors()).toEqual([])
     expect(accessors.getQueryAccessor()).toBe(noopQueryAccessor)
   })
 
-  it("remove() handle null accessor", ()=> {
-    let accessors = new AccessorManager()
-    expect(()=> { accessors.remove(undefined)}).not.toThrow()
+  it('remove() handle null accessor', () => {
+    const accessors = new AccessorManager()
+    expect(() => {
+      accessors.remove(undefined)
+    }).not.toThrow()
   })
 
-  it("getState()", ()=> {
-    this.accessor1.state = new ValueState("a1state")
-    this.accessor4.state = new ValueState("a4state")
+  it('getState()', () => {
+    this.accessor1.state = new ValueState('a1state')
+    this.accessor4.state = new ValueState('a4state')
     expect(this.accessors.getState()).toEqual({
-      p1:"a1state", p4:"a4state"
+      p1: 'a1state',
+      p4: 'a4state'
     })
   })
 
-  it("setState()", ()=> {
+  it('setState()', () => {
     this.accessors.setState({
-      p2:"a2state", p3:"a3state"})
+      p2: 'a2state',
+      p3: 'a3state'
+    })
     expect(this.accessor1.state.getValue()).toBe(null)
-    expect(this.accessor2.state.getValue()).toBe("a2state")
-    expect(this.accessor3.state.getValue()).toBe("a3state")
+    expect(this.accessor2.state.getValue()).toBe('a2state')
+    expect(this.accessor3.state.getValue()).toBe('a3state')
     expect(this.accessor4.state.getValue()).toBe(null)
   })
 
-  it("notifyStateChange", ()=> {
-    let oldState = {}
-    spyOn(PaginationAccessor.prototype, "onStateChange")
+  it('notifyStateChange', () => {
+    const oldState = {}
+    spyOn(PaginationAccessor.prototype, 'onStateChange')
     this.accessors.notifyStateChange(oldState)
-    expect(PaginationAccessor.prototype.onStateChange)
-      .toHaveBeenCalledWith(oldState)
-    expect(PaginationAccessor.prototype.onStateChange["calls"].count())
-      .toBe(4)
+    expect(PaginationAccessor.prototype.onStateChange).toHaveBeenCalledWith(oldState)
+    expect(PaginationAccessor.prototype.onStateChange['calls'].count()).toBe(4)
   })
 
-  it("buildSharedQuery()", ()=> {
-    let query = new ImmutableQuery()
-    this.accessor1.buildSharedQuery = query => query.setSize(25)
-    this.accessor2.buildSharedQuery = query => query.setSize(26)
+  it('buildSharedQuery()', () => {
+    const query = new ImmutableQuery()
+    this.accessor1.buildSharedQuery = (query) => query.setSize(25)
```

Comment:
It's a bit confusing because callback variable shadowing test scope variable.
You can get warning about this by enabling rule `no-shadow` 
Usually, it leads to less bugs and less confusing code, but sometimes you stuck into code like
```
const {someValue} = this.state
const updateSomeValue = (newSomeValue) => this.setState({someValue: newSomeValue})
```